specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2239 @@
+            ]
+          }
+        }
+      }
+    },
+    "SummaryResultItem": {
+      "type": "object",
+      "properties": {
+        "aspect": {
+          "type": "string"
+        },
+        "text": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "aspect",
+        "text"
+      ]
+    },
+    "AnalyzeConversationJobState": {
+      "description": "Contains the status of the analyze conversations job submitted along with related statistics.",
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/JobState"
+        },
+        {
+          "$ref": "#/definitions/ConversationTasksState"
+        },
+        {
+          "$ref": "#/definitions/AnalyzeConversationJobStatistics"
+        }
+      ]
+    },
+    "AnalyzeConversationJobStatistics": {
+      "description": "Contains the statistics for the job submitted.",
+      "properties": {
+        "statistics": {
+          "$ref": "#/definitions/ConversationRequestStatistics"
+        }
+      },
+      "type": "object"
+    },
+    "ConversationTasksState": {
+      "description": "Contains the state for the tasks being executed as part of the analyze conversation job submitted.",
+      "properties": {
+        "tasks": {
+          "properties": {
+            "completed": {
+              "description": "Count of tasks completed successfully.",
+              "type": "integer"
+            },
+            "failed": {
+              "description": "Count of tasks that failed.",
+              "type": "integer"
+            },
+            "inProgress": {
+              "description": "Count of tasks in progress currently.",
+              "type": "integer"
+            },
+            "total": {
+              "description": "Total count of tasks submitted as part of the job.",
+              "type": "integer"
+            },
+            "items": {
+              "description": "List of results from tasks (if available).",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AnalyzeConversationResult"
+              }
+            }
+          },
+          "required": [
+            "total",
+            "completed",
+            "failed",
+            "inProgress"
+          ],
+          "type": "object"
+        }
+      },
+      "required": [
+        "tasks"
+      ],
+      "type": "object"
+    },
+    "AnalyzeConversationJobResult": {
+      "type": "object",
+      "description": "Container for results for all tasks in the conversation job.",
+      "discriminator": "kind",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationResultsKind"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/TaskState"
+        },
+        {
+          "$ref": "common.json#/definitions/TaskIdentifier"
+        }
+      ],
+      "required": [
+        "kind"
+      ]
+    },
+    "ConversationResultBase": {
```

Comment:
shouldn't we put that in common?

Reply:
Discussed offline. Since this is not shared across TA and conversations API not adding to commons.json